source/boards/boards-settings.rst

```diff
@@ -0,0 +1,17 @@
+Settings
+========
+
+Set language
+------------
+
+To set your language on boards, select the gear icon next to your profile avatar, then go to **Set language** to apply your language settings. Please note: language settings in Boards are independent from language settings in Channels.
+
+Emoji icons
+-----------
+
+To enable or disable random emoji icons for your board and cards, select the gear icon next to your profile avatar, then toggle **Random icons on or off**.
```

Comment:
boards? (plural) or board (singular)?

Once you've added the image asset to the source page, you can add the inline reference to the Gear icon here too.